### ISSUE TITLE
How to 'trigger' mpd widget to display status

### ISSUE THREAD

**Issue by ngoonee** [lcpz/lain#449]
I'm using awesome v4.3 (Too long) on Lua 5.3.5.

How would I trigger the mpd widget's status? I've been trying various permutations of theme.mpd.show() | awesome-client but can't seem to get anything to happen.

**Comment by lcpz** [comment]
[This](https://github.com/lcpz/awesome-copycats/blob/master/themes/blackburn/theme.lua#L117-L134) is an example of how to configure the widget, and [this](https://github.com/lcpz/awesome-copycats/blob/master/rc.lua.template#L474-L511) is an example of some keybindings.

You have to use `theme.mpd.update()` within `theme.lua`, and `beautiful.mpd.update()` within `rc.lua`.

**Comment by ngoonee** [comment]
Thanks, took a few weeks to try different things, since my config was almost exactly what you linked @lcpz 

I don't think this works. theme.mpd.update() doesn't do anything for me. If I happen to do os.execute("mpc next") before it, then the notification shows up, but the same thing happens if I use mpc next in the terminal, so theme.mpd.update() is not doing anything.

Now that I re-read, I realised it may be an error in how I've described things. I'm looking to trigger the naughty notification (which pops up every time a new song plays) at any time.

**Comment by lcpz** [comment]
> Can the same be done with mpd?

No, `mpd` does not have a `notify` function. I will add it ASAP.
